modules/nw-multus-tap-object.adoc

```diff
@@ -81,7 +81,66 @@ The following example configures an additional network named `mynet`:
 }
 ----
 
+[id="nw-multus-enable-container_use_devices_{context}"]
+
+== Setting SELinux boolean for the TAP CNI plugin
+
+To create the tap device with the `container_t` SELinux context, enable the `container_use_devices` boolean on the host by using the Machine Config Operator (MCO).
+
+.Prerequisites
+
+* You have installed the OpenShift CLI (`oc`).
+
+.Procedure
+
+. Create a new YAML file named for example `setsebool-container-use-devices.yaml` with the following details:
++
+[source, yaml]
+----
+apiVersion: machineconfiguration.openshift.io/v1
```

Comment:
This yaml is invalid. When I apply it I get:
`error converting YAML to JSON: yaml: line 24: did not find expected key`
Here is the same valid yaml

```
apiVersion: machineconfiguration.openshift.io/v1
kind: MachineConfig
metadata:
  labels:
    machineconfiguration.openshift.io/role: worker
  name: 99-worker-setsebool
spec:
  config:
    ignition:
      version: 3.2.0
    systemd:
      units:
      - enabled: true
        name: setsebool.service
        contents: |
          [Unit]
          Description=Set SELinux booleans
          Before=kubelet.service

          [Service]
          Type=oneshot
          ExecStart=/usr/sbin/setsebool container_use_devices=on
          RemainAfterExit=true

          [Install]
          WantedBy=multi-user.target graphical.target


```